client/utils/serde/clubs-and-events.ts

```diff
@@ -1,82 +1,83 @@
-import moment from 'moment-timezone'
-import { Club, Event } from '../../utils'
+export {}
+// import moment from 'moment-timezone'
+// import { Club, Event } from '../../utils'
 
-// TODO: remove these duplicated types from the backend with shared api types
-export interface BasicEventDTO {
-  name: string
-  startTime: string
-  endTime: string
-  bannerImage?: string
-  tags: Array<string>
-}
+// // TODO: remove these duplicated types from the backend with shared api types
```

Comment:
Since this PR removes `Clubs` and `Events`, this compiler complains about the whole module. Commenting for now because we still want to add de/serialization to API types once we get that hooked up.